rdr_service/model/bq_genomics.py

```diff
@@ -362,21 +362,36 @@ class BQGenomicGCValidationMetricsSchema(BQSchema):
     site_id = BQField('site_id', BQFieldTypeEnum.STRING, BQFieldModeEnum.NULLABLE)
     chipwellbarcode = BQField('chipwellbarcode', BQFieldTypeEnum.STRING, BQFieldModeEnum.NULLABLE)
     idat_green_received = BQField('idat_green_received', BQFieldTypeEnum.INTEGER, BQFieldModeEnum.NULLABLE)
+    idat_green_deleted = BQField('idat_green_deleted', BQFieldTypeEnum.INTEGER, BQFieldModeEnum.NULLABLE)
```

Comment:
Can the new "deleted" fields be added to the end of the schema definition instead of in their current place? When making schema changes to these BQ tables, if the new fields are added to the end, we can just rebuild the records in the PDR as the data is updated. Otherwise, we would need to truncate the tables and then rebuild all records.

Reply:
Done